addons/zeus/XEH_preInit.sqf

```diff
@@ -5,7 +5,7 @@ ADDON = false;
 #include "XEH_PREP.hpp"
 
 if (isServer) then {
-    ["ace_zeusUnitAssigned", FUNC(handleZeusUnitAssigned)] call CBA_fnc_addEventHandler;
+    [QGVAR(zeusUnitAssigned), FUNC(handleZeusUnitAssigned)] call CBA_fnc_addEventHandler;
```

Comment:
Vanilla already invokes a scripted EH https://github.com/acemod/ACE3/blob/03f3b1084d4e9d14617f159db14a7086b1af8682/addons/zeus/functions/fnc_bi_moduleCurator.sqf#L175

No idea how those work though, I `QGVAR`ed it to not have copied stuff from vanilla.